mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -353,7 +353,7 @@ const styles = StyleSheet.create({
     hintsViewCard: {
         marginTop: -verticalScale(40),
         paddingVertical: 0,
-        maxHeight: verticalScale(400)
```

Comment:
Should the maximum height to be relative to the screen size? Wondering how this would look on different phone sizes, especially Android that has a lot of variety. Also, we're supporting iPad and Chromebook which wouldn't cause this to be adaptive to those screens

Reply:
Oh, should I keep this in?

Reply:
I think for now, we can go with your changes and merge it in, this probably need more broader changes in other screens as well. Just wanted to let you know about things that we should consider when changing sizes

Reply:
Okay, sounds good. It's good to keep in mind as we move into the responsive design for larger screens for sure, thank you!